Rocket.Chat/Controllers/Base/Main/MainSplitViewControllerShortcuts.swift

```diff
@@ -0,0 +1,179 @@
+            subscriptionsViewController?.searchController?.dismiss(animated: true) { [weak self] in
+                self?.chatViewController?.keyboardFrame?.updateFrame()
+            }
+        }
+    }
+
+    @objc func shortcutTogglePreferences(_ command: UIKeyCommand) {
+        subscriptionsViewController?.togglePreferences()
+    }
+
+    @objc func shortcutRoomSearch(_ command: UIKeyCommand) {
+        guard !isPresenting else {
+            return
+        }
+
+        if subscriptionsViewController?.searchBar?.isFirstResponder ?? false {
+            subscriptionsViewController?.searchController?.dismiss(animated: true, completion: nil)
+        } else {
+            subscriptionsViewController?.searchBar?.becomeFirstResponder()
+        }
+    }
+
+    @objc func shortcutSelectServer(_ command: UIKeyCommand) {
+        guard let input = command.input, !isPresenting else {
+            return
+        }
+
+        switch input {
+        case "`":
+            subscriptionsViewController?.toggleServersList()
+        case "n":
+            AppManager.addServer(serverUrl: "")
+        default:
+            guard let position = Int(input), position > 0 else {
+                break
+            }
+
+            let index = position - 1
+
+            if index < DatabaseManager.servers?.count ?? 0 {
+                AppManager.changeSelectedServer(index: index)
+            } else {
+                subscriptionsViewController?.openServersList()
+            }
+        }
+    }
+
+    @objc func shortcutSelectRoom(_ command: UIKeyCommand) {
+        guard
+            let viewController = subscriptionsViewController,
+            let input = command.input
+        else {
+            return
+        }
+
+        if input != "n" && isPresenting {
+            return
+        }
+
+        switch input {
+        case "n":
+            viewController.toggleNewRoom()
+        case "]":
+            viewController.selectNextRoom()
+        case "[":
+            viewController.selectPreviousRoom()
+        default:
+            guard let position = Int(input), position > 0 else {
+                break
+            }
+
+            viewController.selectRoomAt(position - 1)
+        }
+    }
+
+    @objc func shortcutUpload(_ command: UIKeyCommand) {
+        chatViewController?.toggleUpload()
+    }
+
+    @objc func shortcutRoomActions(_ command: UIKeyCommand) {
+        chatViewController?.toggleActions()
+    }
+
+    @objc func shortcutRoomMessageSearch(_ command: UIKeyCommand) {
+        chatViewController?.toggleSearchMessages()
+    }
+
+    @objc func shortcutScrollMessages(_ command: UIKeyCommand) {
+        guard
+            !isPresenting,
+            let input = command.input,
+            let offset = chatViewController?.collectionView?.contentOffset,
+            let maxHeight = chatViewController?.collectionView?.contentSize.height,
+            let collectionView = chatViewController?.collectionView
+            else {
```

Comment:
little indentation issue on the `else` statement

Reply:
Thanks!